src/hb_store_gateway.erl

```diff
@@ -215,22 +216,47 @@ cache_read_message_test() ->
 %% know that the module is not respecting the route list.
 specific_route_test() ->
     hb_http_server:start_node(#{}),
+    %% Define the response we want
+    ID = <<"BOogk_XAI3bvNWnxNxwxmvOfglZt17o4MOVAdPNZ_ew">>,
+    Data = <<"specific_route_testx">>,
+    DefaultResponse = {200, Data},
+    Endpoints = [{<<"/raw/", ID/binary>>, ok, DefaultResponse}],
+    {ok, MockServer, _ServerHandle} = hb_mock_server:start(Endpoints),
+    %% Define configuration, we use a valid gateway to obtain a valid response
+    %% and then mock the raw endpoint to our mockserver.
     Opts = #{
         store =>
             [
                 #{ <<"store-module">> => hb_store_gateway, 
-                   <<"routes">> => [],
-                   <<"only">> => local
```

Comment:
Removed `<<only>> => local` because this will make us load everything necessary from the global configuration, like `preloaded_devices`, HTTP timeout configurations, and others. To test only routes, we don't need this.

Reply:
Interesting... Why would `only => local` do that? Its intention is to avoid looking up `Opts` from the global defaults table.

Reply:
Yes, that is the behavior that is shown here. By not loading the remaining configuration from the global defaults, the following modules, which need access to some of the defaults (like `preloaded_devices`), cannot function properly. 

Because we're testing only a different `routes` configuration, we don't need to add `only => local`. If we add it, it will complain of no `preloaded_devices` being available.